app/views/physical_server/show.html.haml

```diff
@@ -0,0 +1,34 @@
+#main_div
```

Comment:
I bet you don't implement all these show types. Please, limit this file to the absolute minimal needed.